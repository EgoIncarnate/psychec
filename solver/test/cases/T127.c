void f()
{
  parse_long_options (1, usage, 9.9);
  usage (EXIT_FAILURE);
}
