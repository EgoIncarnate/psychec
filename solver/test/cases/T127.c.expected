uintptr_t parse_long_options (int,uintptr_t (*) (int),double) ; 
uintptr_t usage (int) ; 
